dspace-server-webapp/src/test/java/org/dspace/app/rest/AuthorizationRestRepositoryIT.java

```diff
@@ -977,16 +977,18 @@ public void findByObjectInternalServerErrorTest() throws Exception {
             .andExpect(status().isInternalServerError());
     }
 
-    @Test
     /**
-     * Verify that the search by object and feature works properly in allowed scenarios:
```

Comment:
Doc comments are meant to be extracted and formatted by e.g. Javadoc, so they should be marked up such that the formatted output is neat and readable.  In particular, the topic sentence should end with a period and lists should be marked up as such.